### PR TITLE
[Section 2] methods in ruby

### PR DESCRIPTION
**Pull request by Michelle-Lohwt**
- Learning how to use methods in ruby

### CODE REVIEW

**Review by tlkiong — COMMENTED** [review]
Why do you need to do `.to_f` in all the method?
Why not `.to_i`?

**Comment by Michelle-Lohwt** [comment]
If you put `.to_i`, you might not get the correct ans (the system most likely will floor the number before any operation, example: `first = 2.5`, `second = 1`, `multiply(first, second) = 2`)

**Review by tlkiong — CHANGES_REQUESTED** [review]
2 spaces indent?
And no empty spaces that does nothing?

To build a masterpiece, start from the details

**Comment by Michelle-Lohwt** [comment]
What does this mean?

> And no empty spaces that does nothing?

File: sec2_the-ruby-programming-language/2.5_methods.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+def multiply(first, second)
+    first.to_f * second.to_f    # automatically return the last line, 
+                                # hence no need to explicitly call "return"
```
Comment (tlkiong) [inline sec2_the-ruby-programming-language/2.5_methods.rb:3]:
```suggestion
    # automatically return the last line, 
    # hence no need to explicitly call "return"
    first.to_f * second.to_f    
```

Put comments to the top

**Review by tlkiong — APPROVED** [review]

**Pull request merged**